### ISSUE TITLE
Dysfunctional item frames limit and ineffective autoclean (item frames and armor stands)

### ISSUE THREAD

**Issue by Chevels** [Zrips/MobFarmManager#92]
Hello,
while testing we noticed that the item frame limit is not working as it should:
Normally they are limited by area around the player (groupchunk: 2) like all other entities, except that my tests confirm that the limit is by chunk.
```
  GroupChunks: true
  GroupChunksRadius: 2
...
RemoveExisting: true
```

```
ITEM_FRAME:
Use: true
Limit: 10
AutoClean: true
```

With this setting, I can put 10 item frames per chunk.
Then I tested to put 30 while being OP to test the autoclean. To be sure I deoped by following then I waited a bit to see if the surplus would be deleted:
- the autoclean does not delete them
- the /mfm clean command does not either

The limit by chunk group and autoclean seems to work for all other entities, but there is a problem with the armor stands because here too the autoclean does not work on them.
I ran the tests on my production server (_latest MFM version at this time: 3.0.2.3_) and on my pre-prod (_MFM 3.0.2.1_), same results.

**To recap:**
Item frames:
- limit is per chunk instead of group of chunks
- /mfm clean and autoclean does not work on them

Armor stands:
- /mfm clean and autoclean does not work on them (_but the limit per groupchunks works fine_)

**Comment by Chevels** [comment]
Any news ?

**Comment by Zrips** [comment]
Will be looking into it tomorrow

**Comment by Zrips** [comment]
New version will have these fixed
